<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>214</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>131</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineHost">
      <property name="placeholderText">
       <string>host...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineLogin">
      <property name="placeholderText">
       <string>login...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineDBname">
      <property name="placeholderText">
       <string>db_name...</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="linePassword">
      <property name="placeholderText">
       <string>password...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="continueBtn">
      <property name="text">
       <string>Continue</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
